tensorflow_addons/losses/contrastive.py

```diff
@@ -88,17 +92,19 @@ class ContrastiveLoss(tf.keras.losses.Loss):
 
     def __init__(
```

Comment:
When adding types to `__init__` functions, you should add the `@typechecked` decorator. You can look at other classes in the codebase for reference.